envoy/upstream/upstream.h

```diff
@@ -565,12 +565,39 @@ class PrioritySet {
 };
 
 /**
- * All cluster stats. @see stats_macros.h
+ * All cluster config update related stats.
+ * See https://github.com/envoyproxy/envoy/issues/23575 for details. Stats from ClusterInfo::stats()
+ * will be split into subgroups "config-update", "lb", "endpoint" and "upstream", roughly based on
```

Comment:
This comment should describe the current state of affairs assuming this PR is checked in, not express a plan.

AFAICT stats_ no longer exists for upstream; it's now fully partitioned so we can just document that.

Reply:
I decided to rename that field in the next PR. 
I think the comment here is kinda accurate, updated the "upstream" one to reflect the "current" status.